common/primitives/src/schema.rs

```diff
@@ -14,6 +14,9 @@ use utils::*;
 /// Schema Id is the unique identifier for a Schema
 pub type SchemaId = u16;
 
+/// Schema version number
+pub type SchemaVersion = u8;
```

Comment:
**Question:** `u8` or something else?

Since the schema ids are u16 it mades sense to use a simple `u8` but open to changes

Reply:
`u8` seems fine--in the rare cases where the number of versions would exceed that, you'd just have to allocate a new name